app/views/courses/TeacherClassView.coffee

```diff
@@ -312,12 +312,14 @@ module.exports = class TeacherClassView extends RootView
           opacity: 1
         }
       })
-    $('.progress-dot, .btn-view-project-level').each (i, el) ->
-      dot = $(el)
-      dot.tooltip({
+
+    $('.has-tooltip').off('mouseenter')
```

Comment:
Is it required to turn off this event on every render?

Reply:
I don't think so. I was being defensive, however identical functions should override the handler. I'd need to look into the coffeescript that is transpiled to see whether this creates any benefit.